### ISSUE TITLE
API Example for setting alternative key names

### ISSUE THREAD

**Issue by abiresque** [paiden/Nett#69]
Hi, this question is better off being on SE or in Wiki, but I figured this is the best place to ask...

I'm trying to read in a 2D matrix (internal math operations) that is serialized as a jagged array as per #54 
I want to use the same TOML key in the serialized file as the CLR property name in my class. 

According to #5, this is possible. However, I'm unclear on how to use the API to map the TOML key name to the CLR property name. It'll be great if someone could share an example.

EDIT 2: Usage scenario
```
class MyClass
{
    [TomlIgnore]
    public int[,] matrix2D { get; private set; } = new int[,] { {1,2,3}, {4,5,6} };
    public int[][] array2D 
    { 
         get { return Mat2Arr(matrix2D); }
         set { matrix2D = Arr2Mat(value); }
    }
    ...
    private int[][] Mat2Arr(int[,] mat) {...}
    private int[,] Arr2Mat(int[][] arr) {...}
}
...
class Program
{
    static void Main(string[] args)
    {
        // Need help defining the settings correctly
        var settings = Nett.TomlSettings.Create(s => s
            .ConfigureType<MyClass>(?)
    --->  .MapKey("matrix2D").ToProperty("array2D")   <---
         );
        ...
    }
}
```

**Comment by paiden** [comment]
#5 was a feature idea that only got closed (delete was not available yet) but never implemented. 

But with this issue here, you provided the first 'real world' use case  for this feature to me. 

So I may reconsider my decision to not implement this.

**Comment by abiresque** [comment]
Thanks, @paiden for the quick update. I clearly misread #5 there.

For your consideration, I particularly like the way [protobuf-net](https://github.com/mgravell/protobuf-net) handles it via the `ProtoMember(Name = "key")` attribute. It would be very convenient if Nett offered a similar option.

**Comment by paiden** [comment]
Released with 0.11.0
